arch/lib/fs.c

```diff
@@ -64,7 +64,10 @@ int dirtytime_interval_handler(struct ctl_table *table, int write,
 	return -ENOSYS;
 }
 
+#if 0
```

Comment:
if possible, please keep those stub calls into arch/lib: otherwise we may need to track the upstreamed kernel changes all the time.

in this particular case, is it possible to create a stub call for nr_free_zone_pages() and keep untouched nr_free_buffer_pages() ?